spec/integration/pr/test-pr-896.spec.js

```diff
@@ -22,11 +22,11 @@ describe('pr related issues', () => {
         .then(wb2 => {
           const ws2 = wb2.getWorksheet('foo');
           expect(ws2.getCell('A1').value).to.equal(' leading');
-          expect(ws2.getCell('A1').note).to.equal(' leading');
+          expect(ws2.getCell('A1').note.texts[0].text).to.equal(' leading');
```

Comment:
could I please you to move your logic from `note` to `notes`  for backward compatibility purposes?
Now, this PR changes the library's API (only for getting a value of course).  `note` will return the first element of notes. 

In other cases, these changes should be published with version 4 and some migration instruction will be added

Reply:
Thank you @Siemienik I re-modified the logic of `note.js`, and now I just added the corresponding property settings on the original basis, without changing the original way of getting the value. It will always return the value set by the user:).